src/write.c

```diff
@@ -1511,6 +1619,64 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
             }
         }
     }
+
+    // Interleave each sample (layer) of each item.
+    //
+    //   - for each layer
+    //    - for each grid cell
+    //     - write alpha of this layer of this grid cell
+    //     - write color of this layer of this grid cell
+    //
+    // See here for the discussion on alpha coming before color:
+    // https://github.com/AOMediaCodec/libavif/issues/287
+
+    if (useInterleave) {
```

Comment:
> This writes all alpha layers/items before all color layers/items right? Would it be possible to write one alpha layer, one color layer, one alpha layer, one color layer etc.? This would be actual "interleaving", not "putting alpha first". I guess it can be odd when the number of alpha and color layers differs but having one layer of each as soon as possible sounds like the best behavior to me.

done